lib/bot.js

```diff
@@ -40,7 +41,10 @@ function Bot(options) {
 Bot.prototype.attachListeners = function() {
   this.client.addListener('registered', function(message) {
     logger.debug('Registered event: ', message);
-  });
+    this.autoSendCommands.forEach(function(element) {
```

Comment:
Fixed. I imagined the tests would have failed since they didn't have autoSendCommands specified either, my bad.